code/modules/reagents/reagent_containers/food.dm

```diff
@@ -71,7 +71,21 @@
 		reagents.maximum_volume = max(reagents.maximum_volume, volume)
 	return ..()
 
-/obj/item/chems/on_reagent_change()
+// Dummy type used solely for soup bowls/soup spoons.
+/obj/item/food/lump
```

Comment:
a little worried in the future that this will result in you scooping up soup as a food lump on a spoon, then the lump immediately melting because it's liquid at room temperature, soaking you and your spoon in soup. but for now it's probably fine, even if it miiiight be better to use `reagents` in the long run for reagent transfer.